add-screenshot/index.md

```diff
@@ -0,0 +1,28 @@
+## Adding screenshots to my game page
```

Comment:
so are we fine w/ calling them screenshots? I remember there was talk around not calling them screenshots anymore... just wanted to check.